yb-voyager/cmd/templates/migration_assessment_report.template

```diff
@@ -345,15 +365,20 @@
             issuePairs.forEach(pair => table.appendChild(pair));
 
             // Update the sort indicator in the table header
-            const sortIndicator = table.querySelector(`th[onclick="sortTableBy('${field}')"] .sort-indicator`);
+            const sortIndicator = table.querySelector(`th[onclick="sortTableBy('${field}', isPerf)"] .sort-indicator`);
             if (sortIndicator) {
                 sortIndicator.innerHTML = newDirection === 'asc' ? '&#8593;' : '&#8595;'; // &#8593; is up arrow, &#8595; is down arrow
             }
-            updateButtonVisibility();
+            updateButtonVisibility(isPerf);
         }
 
+        function updateButtonVisibilities() {
+            updateButtonVisibility(false);
+            updateButtonVisibility(true);
+        }
```

Comment:
what is this function doing?

Reply:
This is on HTML load to update the expand button state of both the tables